substra/sdk/schemas.py

```diff
@@ -191,6 +194,8 @@ class TraintupleSpec(_Spec):
     rank: Optional[int]
     metadata: Optional[Dict[str, str]]
 
+    compute_plan_attr_name: str = "traintuple_keys"
```

Comment:
What do you think of making these classvars (same as `type_`) ? This way it won't show up in `spec.dict()`.